.github/workflows/node-unit-tests.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [14.x, 16.x]
+        node-version: [15.4.0, 16.x]
```

Comment:
if AbortController is 15.4 and up, we can remove 14 and keep 16 only. We do odd numbers just when they are the latest

Reply:
639223e4

Reply:
i know i said this before, i didn't check the schedule of 14. Isn't there a polyfill or something we can suggest?